SPEC.md

```diff
@@ -43,3 +44,17 @@
 ```
 
 The version number must be valid [semver](https://semver.org). See the [npm docs](https://docs.npmjs.com/misc/semver) for more info.
+
+### `main`
+
+Module ID that is the primary entry point to a package - that is when consumer requests the package i.e. with `require('my-package')` the resolution algorithm should check this field and redirect the request to the specified value. **(Default)** `index.js`
```

Comment:
I feel like the “**(Default)** `index.js`” section should go on its own line, preferably before the description.